### PR TITLE
document cable_gw_hydro.F90

### PR DESCRIPTION
**Pull request by bibivking**
# CABLE

Thank you for submitting a pull request to the CABLE Project.

## Description

Please include a brief summary of the change and list the issues that are fixed.
Please also include relevant motivation and context.

You can link issues by using a supported keyword in the pull request's description or in a commit message:

Fixes #(issue)

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix
- [ ] New or updated documentation

## Checklist

- [ ] The new content is accessible and located in the appropriate section.
- [ ] I have checked that links are valid and point to the intended content.
- [ ] I have checked my code/text and corrected any misspellings

Please add a reviewer when ready for review.

### CODE REVIEW

**Review by rogeredberg — COMMENTED** [review]
Suggestion: change comment to indicate intent for future code relocation/refactoring

File: src/science/gw_hydro/cable_gw_hydro.F90 [hunk 1]
```diff
@@ -33,6 +33,10 @@
 !
 ! ==============================================================================
 
+! MMY this code is not actually used in this branch........
+
+
+
```
Comment (har917) [inline src/science/gw_hydro/cable_gw_hydro.F90:39]:
```suggestion
!! MMY this code is not actually used in this branch........

```

**Review by har917 — CHANGES_REQUESTED** [review]
This needs a double !! to be picked up by FORD and likely doesn't need that many blanks lines afterwards

**Comment by ccarouge** [comment]
@bibivking The only pull request I have from you is this one. But this feels like it is the old one as it changes code under src/. Can you confirm I am correct and that this pull request only contains old work? Are you currently working in the same branch or a new branch?

**Pull request closed without merging**